assignment4/count_islands_test.py

```diff
@@ -0,0 +1,49 @@
+'''
+Created on 23 June 2017
+
+@author: Eva
+'''
+
+import unittest
+from count_islands import count_islands_rec, count_islands_iter
+
+class CountingIslandsTest(unittest.TestCase):
```

Comment:
It would be helpful to have smaller test methods that focus on a specific case.
That way, when a test fails you know immediately what it is meant to test.